examples/features/metadata/server/main.go

```diff
@@ -193,11 +195,11 @@ func (s *server) BidirectionalStreamingEcho(stream pb.Echo_BidirectionalStreamin
 
 func main() {
 	rand.Seed(time.Now().UnixNano())
-	lis, err := net.Listen("tcp", port)
+	lis, err := net.Listen("tcp", fmt.Sprintf(":%d", *port))
 	if err != nil {
 		log.Fatalf("failed to listen: %v", err)
 	}
-	fmt.Printf("server listening at port %v\n", port)
+	fmt.Printf("server listening at port %v\n", *port)
```

Comment:
In case port is zero, print `lis.Addr()`?

Reply:
Done